### ISSUE TITLE
Social share images are messed up

### ISSUE THREAD

**Issue by BBlackwo** [bestofjs/bestofjs#366]
I noticed the icons are messed up recently when sharing in Slack. Looks like the logo alt text is being rendered on top of the logo itself?

# Category

<img width="1200" height="630" alt="Image" src="https://github.com/user-attachments/assets/dab63274-28e6-4c76-95c8-8c41f2a44b63" />

# Project

<img width="1200" height="630" alt="Image" src="https://github.com/user-attachments/assets/a5290128-bec9-4a42-b3b6-2804ae0ee06c" />

**Comment by michaelrambeau** [comment]
Actually I think it's because of lint errors from Biome that asked me to add `<title>` attibutes to SVG files!

```tsx
export function TagIcon({ size = "1.5em" }) {
  return (
    <svg
      stroke="currentColor"
      fill="currentColor"
      strokeWidth="0"
      viewBox="0 0 15 16"
      height={size}
      width={size}
    >
      <title>Tag</title>
      <path
        fillRule="evenodd"
        d="M7.73 1.73C7.26 1.26 6.62 1 5.96 1H3.5C2.13 1 1 2.13 1 3.5v2.47c0 .66.27 1.3.73 1.77l6.06 6.06c.39.39 1.02.39 1.41 0l4.59-4.59a.996.996 0 0 0 0-1.41L7.73 1.73zM2.38 7.09c-.31-.3-.47-.7-.47-1.13V3.5c0-.88.72-1.59 1.59-1.59h2.47c.42 0 .83.16 1.13.47l6.14 6.13-4.73 4.73-6.13-6.15zM3.01 3h2v2H3V3h.01z"
      ></path>
    </svg>
  );
}
```
